ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Overview/WorkloadCvesOverviewPage.tsx

```diff
@@ -37,6 +38,8 @@ const emptyStorage: VulnMgmtLocalStorage = {
 
 function WorkloadCvesOverviewPage() {
     const apolloClient = useApolloClient();
+    const { hasReadWriteAccess } = usePermissions();
+    const canWriteWatchedImages = hasReadWriteAccess('WatchedImage');
```

Comment:
An alternative to consider, mostly taste although somewhat for Find in Files:

```ts
const hasWriteAccessForWatchedImage = hasReadWriteAccess('WatchedImage');
```

Having read or written that so many times recently, `hasWriteAccess` function name would be clearer and worth discussion whether to replace all occurrences in 4.4A sprint.